### ISSUE TITLE
Delete button: When text more then icon width - icon align left instead center.

### ISSUE THREAD

**Issue by wedusa** [jarden-digital/react-native-pincode#130]
This was fixed, but dont work again since 1.21 (1.20.41 is ok)
https://github.com/jarden-digital/react-native-pincode/pull/30

**Comment by Haskkor** [comment]
Sorry for that. 
Fixed in latest version (1.22.2)

Cheers,
Jeremy

**Comment by wedusa** [comment]
I have no changes after the update 1.22.2 🤔
```

 "react": "16.9.0",
 "react-native": "0.61.2",
 "@haskkor/react-native-pincode": "^1.22.2",
```
